lib/template.html

```diff
@@ -206,28 +209,57 @@
     "headline": {{ seo_page_title | jsonify }},
 {% endif %}
 
+{% if seo_author_name %}
+    "author": {{ seo_author_name | jsonify }},
+{% endif %}
+
 {% if seo_page_image %}
-    "image": {{ seo_page_image | jsonify }},
+    {% if page.image.height && page.image.width %}
+      "image": {
+          "@type": "ImageObject",
+          "url": {{ seo_page_image | jsonify }},
+          "height": {{ page.image.height | jsonify }},
+          "width": {{ page.image.width | jsonify }}
+      },
+    {% else %}
+        "image": {{ seo_page_image | jsonify }},
+    {% endif %}
```

Comment:
Create image object (as it needs). Height and width are probably required now for some future thinking. Think AMP where they [require this information](https://www.ampproject.org/docs/reference/components/amp-img) in the markup. 

Will default back if either no width/height are present, or if no image url is present.